src/Features/DiagnosticsTestUtilities/Microsoft.CodeAnalysis.Features.DiagnosticsTests.Utilities.csproj

```diff
@@ -9,23 +9,6 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
this project already was checked in, but not referenced in the Roslyn solution.  I updated it to have hte contents of the Editor version, without the actual references to any editor layers.

Reply:
Ah, my bad. I think I ventured into this rabbit hole for a bit and then backed out, but did not clean up everything.